client/Packages/com.beamable.server/Editor/UI2/Configs/BeamableMicroservicesSettingsProvider.cs

```diff
@@ -159,10 +161,10 @@ public static SerializedObject GetSerializedSettings(string serviceName)
 			var instance = ScriptableObject.CreateInstance<BeamableMicroservicesSettings>();
 			instance.serviceName = serviceName;
 
-			instance.serviceDefinitions = new List<AssemblyDefinitionAsset>();
+			instance.assemblyReferences = new List<AssemblyDefinitionAsset>();
 			foreach (var name in sd.AssemblyDefinitionsNames)
 			{
-				var guids = AssetDatabase.FindAssets($"{name} t:AssemblyDefinitionAsset");

```

Comment:
So, doing `typeof(AssemblyDefinitionAsset).FullName` don't work. The name looks something like `UnityInternals.AssemblyDefinitionAsset`, and if you try to find that in the editor, nothing will pop. So, had to use the name instead of the FullName. Not sure if this fixes the issue you presented in our CR tho, so I'm open to suggestions.